.palantir/revapi.yml

```diff
@@ -1178,6 +1178,9 @@ acceptedBreaks:
       new: "class org.apache.iceberg.Metrics"
       justification: "Java serialization across versions is not guaranteed"
     org.apache.iceberg:iceberg-core:
+    - code: "java.method.addedToInterface"
```

Comment:
we can add the functionality without having to break the API. See my other comment on this